### ISSUE TITLE
Unable to content review

### ISSUE THREAD

**Issue by bjherbison** [mozilla/addons-server#11795]
I am a content reviewer. After yesterday's update I am unable to find any add-on I can approve. I am also unable to reject any recent add-on I can find.

From bug #11792:

kewisch:

I can also confirm B.J.'s comment, content reviewers are not able to reject normal add-ons. This one for example: https://reviewers.addons.mozilla.org/en-US/reviewers/review-content/adobe-flash-player-jdek

Going to https://reviewers.addons.mozilla.org/en-US/reviewers/review/adobe-flash-player-jdek makes it reviewable.

 diox :

The content rejection being impossible is weird. I'm guessing it's only for non-auto approved add-ons, but we should fix that.

Could you file separate issues for those things and close this one ?

**Comment by diox** [comment]
Can confirm this happens with non auto-approved add-ons only. I should have anticipated this, the whole code was built around this - you can access the page as a content reviewer but can't do much afterwards.

It goes deep, because content reviewing is re-using auto-approved actions like `confirm_auto_approved`, which don't really make sense any more if we allow non auto-approved add-ons to be content reviewed...

**Comment by bjherbison** [comment]
Does adding this to you To Do list imply this won't be fixed fast? With the content review list cluttered with add-ons I can't review (and no easy way to tell what I can review) the content review process will be much slower and much more frustrating until this is addressed.

**Comment by diox** [comment]
This is a priority 1 issue, it will get fixed today.

**Comment by AlexandraMoga** [comment]
@diox I've verified with several items pending in the Content review queue on -dev and stage, making sure I was including old add-ons that haven't been auto-approved (like search tools, lang packs and dictionaries), add-ons that have been auto-approved but not content reviewed and, finally, add-ons that have a delay auto-approval flag. For all these examples, I was able to Approve or Reject content. 

The Content review queue also shows now a Last Updated column, instead of the Last Content review column. The only observation here is that the items seem to have a random order, but this is not an urgent issue so it can be addressed later.



![image](https://user-images.githubusercontent.com/31961530/60725662-5a1cb700-9f42-11e9-9586-e8e7e043bb2b.png)
* Note that when you select the Approve content you will now have a two extra fields, similar to the old legacy review page, but they are not mandatory 

I've also noticed that there are a few themes left in the queue, for which the only option is to Approve content. However, I've seen that they will be removed from the Content queue altogether, with another patch. 
https://reviewers.addons-dev.allizom.org/en-US/reviewers/review-content/green-hearts
![image](https://user-images.githubusercontent.com/31961530/60724167-5dae3f00-9f3e-11e9-9d91-5df92ff2d988.png)
